papaparse.js

```diff
@@ -1419,10 +1419,6 @@ License: MIT
 			|| Papa.BAD_DELIMITERS.indexOf(comments) > -1)
 			comments = false;
 
-		// Newline must be valid: \r, \n, or \r\n
-		if (newline !== '\n' && newline !== '\r' && newline !== '\r\n')
```

Comment:
Instead of removing the test you should test that newLine is undefined. In this case the default value `\n` should be used.

This is the reason why all tests are failing. 

P.S: You can run the tests on your computer with `npm test` command.